diff --git a/source/core/slang-performance-profiler.cpp b/source/core/slang-performance-profiler.cpp
index faf06bb9..01aa2177 100644
--- a/source/core/slang-performance-profiler.cpp
+++ b/source/core/slang-performance-profiler.cpp
@@ -35,7 +35,7 @@ namespace Slang
             {
                 out << func.key << ": \t";
                 auto milliseconds = std::chrono::duration_cast< std::chrono::milliseconds >(func.value.duration);
-                out << func.value.invocationCount << "\t" << milliseconds.count() << "ms\n";
+                out << func.value.invocationCount << "\t" << static_cast<double>(milliseconds.count()) << "ms\n";
             }
         }
         virtual void clear() override
