diff --git a/source/core/slang-dictionary.h b/source/core/slang-dictionary.h
index 79f6dee3..fcca1160 100644
--- a/source/core/slang-dictionary.h
+++ b/source/core/slang-dictionary.h
@@ -1,7 +1,7 @@
 #ifndef SLANG_CORE_DICTIONARY_H
 #define SLANG_CORE_DICTIONARY_H
 
-#include "../../external/unordered_dense/include/ankerl/unordered_dense.h"
+#include <ankerl/unordered_dense.h>
 #include "slang-common.h"
 #include "slang-exception.h"
 #include "slang-hash.h"
diff --git a/source/core/slang-hash.h b/source/core/slang-hash.h
index ebe3d197..9c5d1242 100644
--- a/source/core/slang-hash.h
+++ b/source/core/slang-hash.h
@@ -1,7 +1,7 @@
 #ifndef SLANG_CORE_HASH_H
 #define SLANG_CORE_HASH_H
 
-#include "../../external/unordered_dense/include/ankerl/unordered_dense.h"
+#include <ankerl/unordered_dense.h>
 #include "slang-math.h"
 #include "slang.h"
 
